### ISSUE TITLE
Structure type sections not being created when running migration

### ISSUE THREAD

**Issue by kmorinaka** [Firstborn/Craft-CMS-Migration-Manager#21]
Thanks for this plugin! It has mostly worked for fields, entries and single type sections, but when running a migration that creates a structure type section, the section doesn't get created and there is no error. Have you run into this before?

**Comment by danprat92** [comment]
Ran into this issue when i was using the plugin.  The problem is in MigrationManager_SectionService.php -> createMode and it happens when  nestedUrlFormat is null or empty because the maxLevel property is not being set anywhere. The solution to this could be adding the following code
```   
        if ($section->type == SectionType::Structure) {
            $section->maxLevels = $data['maxLevels'];
        }
```
Right after this lines 
```
        if ($section->hasUrls) {
            $section->template = $data['template'];
        } else {
            $section->template = $data['template'] = null;
        }
```
Also this line 
```
 'nestedUrlFormat' => $locale['nestedUrlFormat'])
```
Could be changed to
```                 
 'nestedUrlFormat' => !empty($locale['nestedUrlFormat'])? $locale['nestedUrlFormat'] : '',
```
to avoid any errors with the null value as craft SectionsController.php does it similar.

**Comment by fb-derrick-grigg** [comment]
@kmorinaka I had not seen that error but I will look into it. @danprat92 thanks for the fix.

**Comment by fb-derrick-grigg** [comment]
@kmorinaka @danprat92 this has been fixed in 1.0.8.8
